proxy/hdrs/CMakeLists.txt

```diff
@@ -46,3 +46,26 @@ target_link_libraries(hdrs
     PRIVATE
         ts::inkevent
 )
+
+add_executable(test_proxy_hdrs
+    unit_tests/test_HdrHeap.cc
+    unit_tests/test_Hdrs.cc
+    unit_tests/test_HdrUtils.cc
+    unit_tests/test_HdrHeap.cc
+    unit_tests/test_HeaderValidator.cc
+    unit_tests/test_Huffmancode.cc
+    unit_tests/test_mime.cc
+    unit_tests/test_URL.cc
+    unit_tests/unit_test_main.cc
+)
+target_link_libraries(test_proxy_hdrs PRIVATE ts::hdrs ts::tscore ts::inkevent catch2::catch2)
+add_test(NAME test_proxy_hdrs COMMAND test_proxy_hdrs)
+
+add_executable(test_proxy_hdrs_xpack
+    XPACK.cc
+    HuffmanCodec.cc
+    unit_tests/test_XPACK.cc
+)
+target_include_directories(test_proxy_hdrs_xpack PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_link_libraries(test_proxy_hdrs_xpack PRIVATE ts::tscore libswoc catch2::catch2)
```

Comment:
Also depends on tscpputil. Doesn't depend on libswoc.

Reply:
tscore needs libswoc but doesn't have it PUBLIC

Reply:
It almost certainly should be PUBLIC.